### ISSUE TITLE
Monitor State sync with each build

### ISSUE THREAD

**Issue by faddat** [ovrclk/akash#1367]
**Is your feature request related to a problem? Please describe.**
State sync is cool, but unreliable.

**Describe the solution you'd like**
Add a github action that tests state sync with each software change.  Make it a first class citizen.

**Describe alternatives you've considered**
I have tried many ways to sync nodes, and they are all much too slow and much too stateful.

**Additional context**
#1366 is a first stab at this.  It is not expected to pass all the time, but instead to give us an idea of how it's going.

**Comment by boz** [comment]
I love the idea, and am frustrated by state-sync's reliability as well.  We're planning on doing some work to alleviate the state-sync pains, and it'd be great to have this monitoring in place before that.  

That said, I don't think it should be in this repo.  I'll make a `network-status` repo for this and other monitors, which we can run on a cron schedule.  That work?

**Comment by hydrogen18** [comment]
I agree with @boz. If this is just going to be an effort to proactively test the health of the software & the network, maybe we should create another repo? It can live independently of this. We should not be redesigning or reworking state-sync each update. So the test would stay close to the same without needing much in the way of adjustment each update.

**Comment by faddat** [comment]
That makes sense and I'm happy to add the former PR over there.
